code/modules/events/shittening.dm

```diff
@@ -0,0 +1,321 @@
+/datum/round_event_control/shittening
+	name = "Shitty Suggestion Activation"
+	typepath = /datum/round_event/shittening
+	weight = 0
+	earliest_start = INFINITY //a REAL failsafe against it happening ever without admin intervention
+	max_occurrences = 1
+
+/datum/round_event/shittening
+	announceWhen = 0
+	startWhen = 1
+
+/datum/round_event/shittening/announce()
+	priority_announce("Unknown, possibly hostile alien lifeforms resembling feces detected aboard [station_name()], please be wary of shitty behaviors.", "Shit Alert")
+
+//IT BEGINS
+/datum/round_event/shittening/start()
+	ticker.mode.shitty = 1
+	world << "<span class='danger'>The world suddenly feels very shitty.</span>"
+
+
+
+	//Mob changes
+	for(var/mob/M in living_mob_list)
+		if(M.job == "Chaplain")
+			M << "<span class='notice'><b><font size=3>The light of [ticker.Bible_deity_name ? ticker.Bible_deity_name : "the gods"] suffuses you, igniting an inner fire. You are now a paladin!</font></span>"
+			M.verbs += /mob/living/carbon/human/proc/smite_evil
+			M.say("PRAISE")
+		if(M.job == "Botanist")
+			M << "<span class='notice'>You feel far out, man...</span>"
+			M.verbs += /mob/living/carbon/human/proc/summon_dank_blade
+		M << "You feel as if you could go to the nearest computer and create a sci-fi game about space stations."
+		if(M.reagents)
+			M.reagents.add_reagent("programming", 10)
+		M.verbs += /client/verb/mentorhelp
+
+
+
+	//Object changes
+	for(var/obj/item/weapon/reagent_containers/food/snacks/faggot/F in world)
+		if(istype(F, /obj/item/weapon/reagent_containers/food/snacks/faggot/deadchat))
+			continue
+		F.visible_message("<span class='deadsay'><b><i>Strange energies suddenly swirl around \the [F], which begins to glow with an eldritch light.</i></b></span>")
+		new /obj/item/weapon/reagent_containers/food/snacks/faggot/deadchat(F.loc)
+		qdel(F)
+
+	/*
+	for(var/obj/machinery/turret/T in global_turret_list)
+		T.say("Firmware update complete: Switching to High Explosive Rounds.")
+		T.lasertype = 7
+	Uncomment for ~~fun~~! - Iamgoofball
+	for(var/obj/machinery/porta_turret/T in global_turret_list)
+		T.say("Firmware update complete: Switching to High Explosive Rounds.")
+		T.projectile = /obj/item/projectile/bullet/gyro
+		T.eprojectile = /obj/item/projectile/bullet/gyro
+	*/
```

Comment:
Why are rocket turrets commented out of the shitty update? They sound brilliantly shite